aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/FinalResultId.java

```diff
@@ -0,0 +1,12 @@
+package aquality.tracking.integrations.core;
+
+public class FinalResultId {
```

Comment:
maybe enum will be better?

Reply:
I beleive enum is overkill here. I just need constants and don't want to define some additional methods.